syntax = "proto3";

import "protos/base.proto";
import "protos/movie.proto";

service Showtime {
    rpc GetMoviesByDate(Date) returns (Schedule) {}
    rpc GetTimes(Empty) returns (stream MovieID) {}
}

message Date {
    string date = 1;
}

message Schedule {
    Date date = 1;
    repeated MovieID movies = 2;
}